### ISSUE TITLE
UniDepth V1 vitl KITTI Eigen Split Finetuning Training Detail

### ISSUE THREAD

**Issue by Jeffrey-Moon** [lpiccinelli-eth/UniDepth#132]
Hello, first of all, thank you for the great work and the well organized code.
I was working on finetuning the v1 model on the KITTI Eigen Split dataset on five GPUs. I simply changed the code a little bit to load the pretrained model from huggingface but all the code is running without an issue. However, I was wondering if you could provide some insights into the training details of finetuning on the KITTI dataset either in the UniDepth V1 paper or the UniDepth V2 paper. My current configurations are identical to the default configurations provided in the repo (pasted below) and it would be great if you could inform about the approximate settings, especially the types of losses used for the training.

<img width="325" height="364" alt="Image" src="https://github.com/user-attachments/assets/d8c0fef0-1c3c-4c54-bd9a-5b9f1cfee602" />

**Comment by furui123** [comment]
> 您好，首先感谢您出色的工作和井井有条的代码。 我正在使用五块 GPU 对 KITTI Eigen Split 数据集上的 v1 模型进行微调。我只是稍微修改了一下代码，加载了来自 huggingface 的预训练模型，但所有代码运行都没有问题。不过，我想知道您是否可以在 UniDepth V1 论文或 UniDepth V2 论文中提供一些关于在 KITTI 数据集上进行微调的训练细节。我目前的配置与代码库（粘贴在下面）中提供的默认配置相同，如果您能提供一些大概的设置，尤其是用于训练的损失函数类型，那就太好了。
> 
> <img alt="图像" width="325" height="364" src="https://private-user-images.githubusercontent.com/57072299/471681769-d8c0fef0-1c3c-4c54-bd9a-5b9f1cfee602.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTc0MDM1NzUsIm5iZiI6MTc1NzQwMzI3NSwicGF0aCI6Ii81NzA3MjI5OS80NzE2ODE3NjktZDhjMGZlZjAtMWMzYy00YzU0LWJkOWEtNWI5ZjFjZmVlNjAyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTA5VDA3MzQzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYyNTc5ZGY4Mzk0ZDQ5MjhiZTRlNWVmMzhjMTA4ODM1M2M4ZTJhNWQwNDNjYmJjY2I1ZDdmODg2YTVmZTEzNzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.bpKlAhXeQzsyA-sX4-6mtzCjCi4nxwrUbkmrTnXJvyM">

Hello, regarding the type of loss function used for training and the related settings you mentioned, could you please share the specific details? I also want to use KITTI for fine-tuning. Thanks again.

**Comment by furui123** [comment]
@ @Jeffrey-Moon Hello, regarding the type of loss function used for training and the related settings you mentioned, could you please share the specific details? I also want to use KITTI for fine-tuning. Thanks again.
